### PR TITLE
hide table only available to private preview

### PR DESCRIPTION
**Pull request by nickt444**
Change(s):
   - Updated workbook to hide query results if required tables don't exist

   Reason for Change(s):
   - Users can join private preview (instructions provided) to see reports that rely on TVM data in Sentinel
   - if they haven't joined private preview, the corresponding queries that would otherwise throw errors are hidden

   Version updated:
   - No

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by v-sabiraj** [comment]
@nickt444, please update the version in [Metadata ](https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/WorkbooksMetadata.json)file for the workbook, thanks.

**Comment by nickt444** [comment]
@v-sabiraj I'm not sure why workbook version is failing validation. This is the 3rd increase in version number

File: Workbooks/AcscEssential8.json [hunk 1]
```diff
@@ -3474,7 +3583,7 @@
       }
     }
   ],
-  "fallbackResourceIds": [],  
-  "fromTemplateId": "sentinel-AcscEssential8Workbook",
+  "fallbackResourceIds": [],
+  "fromTemplateId": "sentinel-AcscEssential8",
```
Comment (v-sabiraj) [inline Workbooks/AcscEssential8.json:3587]: @nickt444, maybe the templateid is changed, can you please revert the change, thanks.

**Comment by v-sabiraj** [comment]
@nickt444, also there is a PR https://github.com/Azure/Azure-Sentinel/pull/7479 with changes in same file, can you please collate in this same PR so we can close that, thanks.

File: Workbooks/AcscEssential8.json (hunk 1 above)
Comment (nickt444) [reply Workbooks/AcscEssential8.json]: done

**Comment by nickt444** [comment]
@v-sabiraj I keep incrementing version number but it still fails validation. I integrated the change from PR 7479 so that can be closed, and reverted back to original fromTemplateId.

**Review by v-dvedak — APPROVED** [review]
